xcodeproj/internal/files.bzl

```diff
@@ -62,6 +62,37 @@ def parsed_file_path(path):
     else:
         return project_file_path(path)
 
+_FOLDER_TYPE_EXTENSIONS = {
+    ".bundle": None,
+    ".docc": None,
+    ".framework": None,
```

Comment:
`.xcframework` too?

Reply:
No, since the way that bazel handles xcframeworks is to process itself and expose the underlaying frameworks. So in the generated project we need to treat xcframeworks as simple folders. I have a PR soon that will have an example usecase of importing xcframeworks.